testutil/network/sessionnet/genesis.go

```diff
@@ -0,0 +1,104 @@
+package sessionnet
+
+import (
+	"fmt"
+	"testing"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/stretchr/testify/require"
+
+	apptypes "github.com/pokt-network/poktroll/x/application/types"
+	sharedtypes "github.com/pokt-network/poktroll/x/shared/types"
+	suppliertypes "github.com/pokt-network/poktroll/x/supplier/types"
+)
+
+// ConfigureDefaultSupplierModuleGenesisState generates and populates the in-memory
+// network's application module's GenesisState object with a given number of suppliers,
+// each of which is staked for a unique service. It returns the genesis state object.
+func (memnet *inMemoryNetworkWithSessions) configureSupplierModuleGenesisState(t *testing.T) *suppliertypes.GenesisState {
```

Comment:
```suggestion
func (memnet *inMemoryNetworkWithSessions) ConfigureSupplierModuleGenesisState(t *testing.T) *suppliertypes.GenesisState {
```
Comment is exported but not function double check these and others

Reply:
Nice catch - thanks!